spec/requests/list_all_articles_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+describe "Get /api/articles" do
+    let(:category){create(:category)}
+    let!(:articles){ 5.times {FactoryBot.create(:article)}}
```

Comment:
Create an article that is not published to make sure that you are only getting approved articles.

Reply:
done